.github/workflows/dashboard-pr.yml

```diff
@@ -31,18 +31,13 @@ jobs:
       - name: Update Files
         run: |
           web/gui/bundle_dashboard.py ${{ github.event.inputs.dashboard_version }}
-      - name: Commit Changes
-        uses: swinton/commit@v2.x
-        env:
-          GH_TOKEN: ${{ secrets.NETDATABOT_GITHUB_TOKEN }}
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request
```

Comment:
Shouldn't this be pinned to a specific version, eg. `uses: peter-evans/create-pull-request@v3`?